pandas/tests/frame/methods/test_to_csv.py

```diff
@@ -251,7 +251,7 @@ def make_dtnat_arr(n, nnat=None):
             df = DataFrame(dict(a=s1, b=s2))
             df.to_csv(pth, chunksize=chunksize)
 
-            recons = self.read_csv(pth)._convert(datetime=True, coerce=True)
```

Comment:
interesting we must have removed all the internal uses of `_convert` (or we should)